$OpenBSD: patch-configure,v 1.12 2010/11/18 16:44:33 jasper Exp $

Stupid configure is stupid, just work around the mess by setting
the values we know are correct.

--- configure.orig	Thu Nov 18 17:08:08 2010
+++ configure	Thu Nov 18 17:11:23 2010
@@ -20110,6 +20110,13 @@ $as_echo "$as_me: BDB backend not used" >&6;}
 	fi
     fi
 
+# XXX
+bdb_found=yes
+BDB_CFLAGS=-I${bdbdir}/include/db4
+BDB_LIBS="-L${bdbdir}/lib -ldb"
+BDB_LIB="-ldb"
+BDB_DIR="$bdbdir"
+
      if test "$bdb_found" = "yes"; then
   BDB_TRUE=
   BDB_FALSE='#'
